providers/catalog.go

```diff
@@ -2148,4 +2149,29 @@ var catalog = CatalogType{ // nolint:gochecknoglobals
 			Write:     false,
 		},
 	},
+
+	// Hunter Configuration
+	Hunter: {
+		AuthType: ApiKey,
+		BaseURL:  "https://api.hunter.io/",
+		// For 6sense, the header needs to be 'Authorization: Token {your_api_key}'
+		ApiKeyOpts: &ApiKeyOpts{
+			Type: InQuery, // Can also be InQuery
+			// HeaderName: "X-API-KEY",
```

Comment:
Should probably clean this up before merging

Reply:
will do